CHANGES/639.packaging.rst

```diff
@@ -0,0 +1 @@
+Use SPDX license identifier for project metadata.
```

Comment:
Please change the writing style from a "TODO-list item" to "X changed" or "X is now Y".